package.json

```diff
@@ -7,16 +7,17 @@
     "@material-ui/core": "3.5.1",
     "@material-ui/icons": "3.0.1",
     "@material-ui/lab": "3.0.0-alpha.23",
+    "firebase": "^5.5.9",
```

Comment:
Let's fix the package version and remove ^.
Same for react-redux. What do you think?